httpd/handler.go

```diff
@@ -52,8 +52,9 @@ type Handler struct {
 	requireAuthentication bool
 	version               string
 
-	Logger     *log.Logger
-	WriteTrace bool // Detailed logging of write path
+	Logger            *log.Logger
+	HTTPAccessLogging bool // Log every HTTP access.
```

Comment:
Can we just make this something like `EnableLogging` or `LoggingEnabled`?  This name is confusing when I don't see it in context.